[
  {
    "id": "23002289",
    "etr": "ETR-NULL",
    "etrConfidence": null,
    "cluster": false,
    "comments": "We are aware of outages in this area and will provide an estimated time of restoration as soon as possible.",
    "cause": "Unable to assess due to conditions",
    "numberOut": 1,
    "custAffected": 4,
    "crewStatus": null,
    "startTime": "2023-01-12T14:09:21Z",
    "latitude": 37.77933,
    "longitude": -84.84816,
    "source": "https://kubra.io/cluster-data/300/aa3e643b-d606-4bc6-944f-a2f22f0d5879/2067b372-1605-4513-8d36-6da0d4d42ee8/public/cluster-3/0320003.json"
  }
]